src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs

```diff
@@ -609,8 +609,8 @@ internal override void OnItemClicked(int clickedIndex, VirtualKeyModifiers modif
 
 			void SingleSelectionCase()
 			{
-
-				if (ItemsSource is ICollectionView collectionView)
+				//Making sure that the ItemsSource is a CollectionView and not ISelectionInfo as it breaks the selection logic
```

Comment:
```suggestion
				// The `ISelectionInfo` -handled directly by the `ItemsSource`- has precedence over the `ICollectionView`. If an object implements both interfaces, as WinUI, we make sure to use only the `ISelectionInfo` to avoid conflicting interaction.
```